api/sales_dashboard/views.py

```diff
@@ -53,6 +54,7 @@
 DEFAULT_ORGANISATION_SORT_DIRECTION = "DESC"
 
 email_regex = re.compile(r"^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$")
+domain_regex = re.compile(r"^[a-z0-9.-]+\.[a-z]{2,}$")
```

Comment:
I'd prefer if this domain lookup was made explicit, for example by searching for `@example.com` instead of `example.com`.

Contrived example: imagine you're looking for an organisation named `apple.com` and the users all have email domains of `icloud.com`. You would have to search for `apple` or a similar query instead.

By making this explicit we can arbitrarily search for email domains without caring about possible conflicts with organisation names.

Reply:
Let's see if that becomes and issue. I think the current implementation is better UX than what you're suggesting because I imagine people would want to search just the domain, and would likely forget they have to prepend `@` to search on the domain.